Clueless/Clueless/templates/play.html

```diff
@@ -422,7 +422,7 @@
                     //alert('No one has won this game...\nSolution set is: ' + idToText[solution_cards["Suspects"][0]]["name"] + ' in the ' + idToText[solution_cards["Rooms"][0]]["name"] + ' with the ' + idToText[solution_cards["Weapons"][0]]["name"]);
                 }
                 else {
-                	showSolutionSet({{player_name | safe}});
+                	showSolutionSet({{ game_winner | safe }});
                     //alert( {{ game_winner | safe }} +' has won the game!!\nSolution set is: ' + idToText[solution_cards["Suspects"][0]]["name"] + ' in the ' + idToText[solution_cards["Rooms"][0]]["name"] + ' with the ' + idToText[solution_cards["Weapons"][0]]["name"]);
                 }
                 location.replace("/");
```

Comment:
I think this line is what's redirecting everyone back to the homepage immediately. Can you try commenting this out?
See docs: https://www.w3schools.com/jsref/met_loc_replace.asp